homeassistant/components/flipr/coordinator.py

```diff
@@ -32,6 +32,10 @@ def __init__(
             update_interval=timedelta(minutes=15),
         )
 
+
+class FliprDataUpdateCoordinator(BaseDataUpdateCoordinator):
```

Comment:
Can we, maybe in a preliminary PR, add typing to the current coordinator? Please add the `_async_update_data` return type and add that to the `DataUpdateCoordinator` type like so: `class FliprDataUpdateCoordinator(DataUpdateCoordinator[....]):`

Once we split the coordinators and the return type is different, we should make the Base one generic. Check the withings coordinator for an example

Reply:
Typed it : the FliprDataUpdateCoordinator and the FliprHubDataUpdateCoordinator have the same type since their _async_update_data return is of the same type.